pocs/tests/utils/test_utils.py

```diff
@@ -9,12 +9,34 @@
 from pocs.utils import listify
 from pocs.utils import load_module
 from pocs.utils import CountdownTimer
-from pocs.utils.error import NotFound
+from pocs.utils import error
 from pocs.camera import list_connected_cameras
 
 
+def test_error(capsys):
+    with pytest.raises(Exception) as e_info:
```

Comment:
Should this be PanError instead of Exception? i.e. why test for Exception being raised?